stationkb/pt/connections/bus/pt-ml-ae.html

```diff
@@ -35,4 +35,4 @@
     <ul>
         <li><b>1</b> City Center</li>
     </ul>
-</p
\ No newline at end of file
+</p
```

Comment:
The `</p>` is not correct here either

Also, what did you change exactly?

Reply:
<li><b>705</b> Estação Oriente (Interface) ⇄ Estação Roma-Areeiro</li>
 -		<li><b>708</b> Martim Moniz ⇄ Parque das Nações Norte</li>
 -		<li><b>717</b> Praça do Chile ⇄ Fetais</li>
 +	<li><b>708</b> Martim Moniz ⇄ Parque das Nações Norte</li>
this

Reply:
i just put the lines aligned

Reply:
oh ok, then take the opportunity to fix the HTML tags which are missing the `>` and I'll merge this

(just add more commits to your master branch)